content/docs/user-guide/serving/streamlit.md

```diff
@@ -0,0 +1,58 @@
+# Streamlit
+
+[Streamlit](https://streamlit.io) is an open-source Python library that makes it
+easy to create and share beautiful, custom web apps for machine learning and
+data science.
+
+MLEM can use Streamlit to expose your model to external users with a nice
+auto-generated UI.
+
+## Requirements
+
+```bash
+pip install mlem[streamlit]
+# or
+pip install fastapi uvicorn streamlit streamlit_pydantic
+```
+
+## Examples
+
+### Running Streamlit model server from CLI
+
+```cli
+$ mlem serve streamlit \
+  --model https://github.com/iterative/example-mlem-get-started/models/rf
+⏳️ Loading model from https://github.com/iterative/example-mlem-get-started/tree/main/models/rf.mlem
+Starting streamlit server...
+🖇️  Adding route for /predict
+🖇️  Adding route for /predict_proba
+Checkout openapi docs at <http://0.0.0.0:8080/docs>
+INFO:     Started server process [42223]
+INFO:     Waiting for application startup.
+INFO:     Application startup complete.
+INFO:     Uvicorn running on http://0.0.0.0:8080 (Press CTRL+C to quit)
+
+  You can now view your Streamlit app in your browser.
+
+  URL: http://0.0.0.0:80
```

Comment:
Remove all output from the sample? We didn't add it in https://mlem.ai/doc/user-guide/serving/fastapi, for example.